lib/postmark/error.rb

```diff
@@ -68,6 +72,12 @@ def retry?
     end
   end
 
+  class InvalidEmailAddressError < ApiInputError
+    def initialize(*args)
+      super
+    end
+  end
```

Comment:
```suggestion
  class InvalidEmailAddressError < ApiInputError; end
```

Reply:
I see you marked this as resolved but you didn't apply this suggestion. What I meant here is that this constructor is not necessary as it's just calling `super` and we could remove it.

Reply:
ah sorry, missed that for some reason in GH, applied the change now